src/workerd/jsg/resource.c++

```diff
@@ -24,6 +25,28 @@ void exposeGlobalScopeType(v8::Isolate* isolate, v8::Local<v8::Context> context)
   KJ_ASSERT(check(global->Set(context, name, constructor)));
 }
 
+void polyfillSymbols(jsg::Lock& js, v8::Local<v8::Context> context) {
+  js.withinHandleScope([&]() {
+    JsObject obj(context->Global());
+
+    auto symbol = KJ_ASSERT_NONNULL(obj.get(js, "Symbol").tryCast<JsObject>());
+
+    KJ_ASSERT(!symbol.has(js, "dispose") && !symbol.has(js, "asyncDispose"),
```

Comment:
Probably only needs to be checked in debug builds?

```suggestion
    KJ_DASSERT(!symbol.has(js, "dispose") && !symbol.has(js, "asyncDispose"),
```

Reply:
f8ded118